src/comp.rs

```diff
@@ -24,31 +24,6 @@ pub enum Message {
 	NewPrivilegedClient { count: usize },
 }
 
-pub fn create_privileged_socket(
-	sockets: &mut Vec<UnixStream>,
-	env_vars: &[(String, String)],
-) -> Result<(Vec<(String, String)>, OwnedFd)> {
-	// Create a new pair of unnamed Unix sockets
-	let (comp_socket, client_socket) =
-		UnixStream::pair().wrap_err("failed to create socket pair")?;
-	// Push one socket to the list of sockets we were passed
-	sockets.push(comp_socket);
-	// Turn the other socket into a non-blocking fd, which we can pass to the child
-	// process
-	let client_fd = {
-		let std_stream = client_socket
-			.into_std()
-			.wrap_err("failed to convert client socket to std socket")?;
-		std_stream
-			.set_nonblocking(true)
-			.wrap_err("failed to mark client socket as non-blocking")?;
-		OwnedFd::from(std_stream)
-	};
-	let mut env_vars = env_vars.to_vec();
-	env_vars.push(("WAYLAND_SOCKET".into(), client_fd.as_raw_fd().to_string()));
-	Ok((env_vars, client_fd))
-}
```

Comment:
I don't see anything else changed referencing this function? Was this just a left-over?